### PR TITLE
Typos and small changes

### PR DESCRIPTION
**Pull request by AStonedApe**
Fixed a few typos and changed "data" to "content" to make it sound less intimidating to non-technical folks.

### CODE REVIEW

**Comment by AStonedApe** [comment]
I hope I did this right, still learning how to use github

File: docs/introduction/about.mdx
Comment (musoke) [inline docs/introduction/about.mdx:1]: Looks like one new typo here

**Review by musoke — CHANGES_REQUESTED** [review]
Thank you! This looks good overall.  Could you fix the one error I highlighted?

**Comment by musoke** [comment]
Did you mean to close this?  You should be able to add one more commit to the same pull request.

**Pull request closed without merging**